sm/tedge_agent/tests/main.rs

```diff
@@ -8,6 +7,7 @@ mod tests {
     use predicates::prelude::*;
 
     #[test]
+    #[ignore] // this test requires sudo
```

Comment:
Ahh, these tests require special runner for cargo.

I use attribute on my fork to run these only when special runner is available.
It can be done using env variables as well, something along the lines of : `CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUNNER="sudo -E"`